CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+# dbt_ad_reporting v1.14.0
+
+[PR #144](https://github.com/fivetran/dbt_ad_reporting/pull/144) includes the following updates:
+
+**61 total changes • 1 possible breaking change (see `ad_reporting__search_report`)**
```

Comment:
thoughts on including all the upstream changes, even the staging models?

Reply:
Including all those changes is quite exhaustive. Maybe we just include a link to the respective release notes and mention the number of breaking changes?